app/services/oa_workflow_service.rb

```diff
@@ -33,5 +33,12 @@ def workflow
     rescue StandardError
       file.update_column(:downloaded, false)
     end
+
+    ActivityInsightOAFile.send_oa_status_to_activity_insight.each do |file|
+      publication = file.publication
+      publication.exported_oa_status_to_activity_insight = true
```

Comment:
I think in order to cover every record in Activity Insight that needs their status updated we will have to make `exported_oa_status_to_activity_insight` an attribute of ActivityInsightOAFile.  For example: if two people upload files in Activity Insight for the same publication.  These will be two separate records in Activity Insight, but could get merged into one in RMD.  So we might end up only updating the status for one of them if we store this with Publication.